XenAdmin/Dialogs/PropertiesDialog.cs

```diff
@@ -115,25 +114,25 @@ private void Build()
         {
             var pool = Helpers.GetPoolOfOne(connection);
 
-            bool is_host = xenObjectCopy is Host;
-            bool is_vm = xenObjectCopy is VM && !((VM)xenObjectCopy).is_a_snapshot;
-            bool is_sr = xenObjectCopy is SR;
+            bool is_host = _xenObjectCopy is Host;
+            bool is_vm = _xenObjectCopy is VM && !((VM)_xenObjectCopy).is_a_snapshot;
+            bool is_sr = _xenObjectCopy is SR;
 
-            bool is_pool = xenObjectCopy is Pool;
-            bool is_vdi = xenObjectCopy is VDI;
-            bool is_network = xenObjectCopy is XenAPI.Network;
+            bool is_pool = _xenObjectCopy is Pool;
+            bool is_vdi = _xenObjectCopy is VDI;
+            bool is_network = _xenObjectCopy is XenAPI.Network;
 
-            bool is_hvm = is_vm && ((VM)xenObjectCopy).IsHVM();
-            bool is_template = is_vm && ((VM)xenObjectCopy).is_a_template;
-            bool is_in_pool = Helpers.GetPool(xenObjectCopy.Connection) != null;
+            bool is_hvm = is_vm && ((VM)_xenObjectCopy).IsHVM();
+            bool is_template = is_vm && ((VM)_xenObjectCopy).is_a_template;
+            bool is_in_pool = Helpers.GetPool(_xenObjectCopy.Connection) != null;
 
             bool is_pool_or_standalone = is_pool || (is_host && !is_in_pool);
 
-            bool wlb_enabled = (Helpers.WlbEnabledAndConfigured(xenObjectCopy.Connection));
+            bool wlb_enabled = (Helpers.WlbEnabledAndConfigured(_xenObjectCopy.Connection));
 
-            bool is_VM_appliance = xenObjectCopy is VM_appliance;
+            bool is_VM_appliance = _xenObjectCopy is VM_appliance;
 
-            bool is_VMSS = xenObjectCopy is VMSS;
+            bool is_VMSS = _xenObjectCopy is VMSS;

```

Comment:
nit picky but since you've renamed variables already let's get rid of snake case here for all of these. ReSharper should make it painless

Reply:
Done;  I also made the checks slightly more efficient.